images/android/v3.8-python-jira-git/Dockerfile

```
@@ -0,0 +1,4 @@
+FROM python:3.8
+LABEL maintainer="Borgalinov Timur <borgalinov@gmail.com>"
+
+RUN pip install click jira PyGithub
```

```suggestion
RUN pip install --no-cache-dir click jira PyGithub
```
https://app.deepsource.com/directory/analyzers/docker/issues/DOK-P1003